experiment_helpers/run_passageReading.m

```diff
@@ -0,0 +1,64 @@
+function run_passageReading()
+% Run Audapter in feedback mode 3 (hear speech and noise) while displaying
+%   a black screen. Before and after running Audapter, display a "Time for a
+%   break" message. 
+%
+% Intended to be used for experiments like cerebAAF and adaptRetest, where
+%   the participant is reading the "rainbow" passage or something similar
+%   and hearing their own speech with unaltered feedback.
+
+% 2022-?? Ben Parrell wrote initial code
+% 2022-10 Chris Naber split out to own function and updated.
+
+
+% Set up displays
+h_fig = setup_exptFigs;
+get_figinds_audapter; % names figs: stim = 1, ctrl = 2, dup = 3;
+
+% set text
+color2display = [1 1 1]; %white
+stimtxtsize = 50;
+
+fprintf('Starting passage-reading section.\nGive instructions, then press any key to START Audapter.\n');
+h_text = draw_exptText(h_fig,.5,.5,'Time for a break.', 'Color',color2display, 'FontSize',stimtxtsize, 'HorizontalAlignment','center');
+pause
+
+% clear screen
+delete_exptText(h_fig,h_text)
+clear h_text
+
+%set up audapter
+audioInterfaceName = 'Focusrite USB'; %SMNG default for Windows 10
+Audapter('deviceName', audioInterfaceName);
+
+% nullify online status tracking/pert config files (use pert field instead)   
+Audapter('ost', '', 0);         
+Audapter('pcf', '', 0);
+
+% set audapter params
+p = getAudapterDefaultParams('female'); % get default params. Gender doesn't matter b/c not perturbing
+p.bShift = 1;
```

Comment:
Why bShift = 1 if "not perturbing"?

Reply:
It could probably be zero. It was 1 in the code I was taking this from. I can test it with 0 if you'd like.

Reply:
Removed these params since they're unnecessary